xml/System.Security.Cryptography.X509Certificates/X509CertificateCollection.xml

```diff
@@ -1107,9 +1107,9 @@
         <Parameter Name="index" Type="System.Int32" Index="0" FrameworkAlternate="netcore-1.0;netcore-1.1;netstandard-1.3;netstandard-1.4;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="index">To be added.</param>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <param name="index">The zero-based index of the element to get or set.</param>
+        <summary>Gets or sets the element at the specified index.</summary>
+        <value>The element at the specified index.</value>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Security.Cryptography.X509Certificates.X509CertificateCollection> instance is cast to an <xref:System.Collections.IList> interface.  

           ]]></format>
         </remarks>
```